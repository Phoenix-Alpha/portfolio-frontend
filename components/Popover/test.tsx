import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { Popover } from '.'

describe('<Popover />', () => {
  const children = 'Test children'

  it('should renders correct <Popover />', () => {
    const wrapper = shallow(<Popover>{children}</Popover>)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
