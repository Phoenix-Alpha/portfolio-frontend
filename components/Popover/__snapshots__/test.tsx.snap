// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Popover /> should renders correct <Popover /> 1`] = `
<div
  className="absolute z-10 w-[80vw] md:w-auto md:right-full md:left-auto md:top-0 md:mr-1"
>
  <div
    className="py-1 px-3 bg-white rounded motion-reduce:animate-none animate-grow-vertical"
    style={
      Object {
        "boxShadow": "0px 2px 8px rgba(12, 8, 21, 0.2)",
      }
    }
  >
    Test children
  </div>
</div>
`;
